snippets/csharp/NostrWalletConnect.cs

```diff
@@ -0,0 +1,89 @@
+using Breez.Sdk.Liquid;
+
+// ANCHOR: event-listener
+public class MyNwcEventListener : NwcEventListener
+{
+    public async Task OnEvent(NwcEvent event)
+    {
+        switch (event.Details)
+        {
+            case NwcEventDetails.Connected:
+                Console.WriteLine("NWC connected");
+                break;
+            case NwcEventDetails.Disconnected:
+                Console.WriteLine("NWC disconnected");
+                break;
+            case NwcEventDetails.PayInvoice payInvoice:
+                Console.WriteLine($"Payment {(payInvoice.Success ? "successful" : "failed")}");
+                break;
+            case NwcEventDetails.ListTransactions:
+                Console.WriteLine("Transactions requested");
+                break;
+            case NwcEventDetails.GetBalance:
+                Console.WriteLine("Balance requested");
+                break;
+        }
+    }
+}
+// ANCHOR_END: event-listener
+
+public class NostrWalletConnect
+{
+    public static async Task NostrWalletConnectAsync()
+    {
+        // ANCHOR: nwc-config
+        var nwcConfig = new NwcConfig
+        {
+            RelayUrls = new[] { "<your-relay-url-1>" },       // Optional: Custom relay URLs (uses default if null)
+            SecretKeyHex = "your-nostr-secret-key-hex"        // Optional: Custom Nostr secret key
+        };
+        
+        var nwcService = new SdkNwcService(nwcConfig);
+        
+        // Add the plugin to your SDK
+        var plugins = new Plugin[] { nwcService };
+        // ANCHOR_END: nwc-config
+
+        // ANCHOR: add-connection
+        var connectionName = "my-app-connection";
+        var connectionString = await nwcService.AddConnectionString(connectionName);
+        // ANCHOR_END: add-connection
+
+        // ANCHOR: list-connections
+        var connections = await nwcService.ListConnectionStrings();
+        // ANCHOR_END: list-connections
+
+        // ANCHOR: remove-connection
+        await nwcService.RemoveConnectionString(connectionName);
+        // ANCHOR_END: remove-connection
+
+        // ANCHOR: event-listener
+        // Event listener class is defined above
+        // ANCHOR_END: event-listener
```

Comment:
You can remove this if you defined this above. Same with the other language snippets